test/button/button-subtle.html

```diff
@@ -0,0 +1,101 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+		<title>subtle button tests</title>
+
+		<script src="../../../@webcomponents/webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="../../../mocha/mocha.js"></script>
+		<script src="../../../chai/chai.js"></script>
+		<script src="../../../@polymer/test-fixture/test-fixture.js"></script>
+		<script src="../../../wct-mocha/wct-mocha.js"></script>
+
+		<script type="module" src="../../components/button/button-subtle.js"></script>
+	</head>
+	<body>
+		<test-fixture id="basic">
+			<template>
+				<d2l-button-subtle text="Subtle Button"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+		<test-fixture id="with-icon">
+			<template>
+				<d2l-button-subtle text="Subtle Button with Icon" icon="d2l-tier1:gear"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+		<test-fixture id="disabled">
+			<template>
+				<d2l-button-subtle disabled text="Disabled Subtle Button"></d2l-button-subtle>
+			</template>
+		</test-fixture>
+
+		<script type="module">
+			import { runAxe } from '../../test-helpers/a11y-helper.js';
+
+			suite('subtle button', function() {
+				let button;
+				suite('basic', function() {
+					setup(function(done) {
+						button = fixture('basic');
+						requestAnimationFrame(function() {
+							done();
+						});
+					});
+					test('should execute onclick when clicked', function(done) {
+						button.addEventListener('click', function() {
```

Comment:
Same as the other Dave mentioned, this could just be `button.addEventListener('click', done);`

Reply:
Same issue, seeing 
```
Error: done() invoked with non-Error: [object MouseEvent]
  Context.<anonymous> at button-subtle.html:44
```
Is there a way to tell the event listener not to automatically pass in the event as a parameter?

Reply:
Oh wow.  Not that I know of.  I guess this would be a recommended use of arrow function if we care.  `button.addEventListener('click', () => { done(); });`

Reply:
Yeah, that would clean it up nicely, but are we ok mixing arrow functions and old syntax?  Looks like you're already mixing in `button-subtle-perceptual.js` anyways, so I can change this whole file to use arrow functions

Reply:
As much as I like the consistency, I think it's ok to mix since there are valid places to use each.  I'd be fine with changing the perceptual ones to be consistent though.